templates/ministerial/ministerial-en.html

```diff
@@ -33,113 +29,24 @@ <h2>Contact information</h2>
-		<section>
-			<h2>Latest</h2>
-			<div class="gc-nws row">
-				<div class="col-md-8">
-					<div class="row">
-						<div class="col-md-6">
-							<a href="#">
-								<img src="../img/news-360x203.png" alt="" class="img-responsive thumbnail">
-								<p>[News title]</p>
-							</a>
-							<p><small class="text-muted"><time>2018-06-05 19:00</time> - Type of news product</small><br />
-							Brief description of the news item.</p>
-						</div>
-						<div class="col-md-6">
-							<a href="#">
-								<img src="../img/news-360x203.png" alt="" class="img-responsive thumbnail">
-								<p>[News title]</p>
-							</a>
-							<p><small class="text-muted"><time>2018-06-05 20:00</time> - Type of news product</small><br />
-							Brief description of the news item.</p>
-						</div>
-					</div>
-				</div>
-				<div class="wb-feeds limit-3 col-md-4">
-					<ul class="feeds-cont list-unstyled lst-spcd">
-						<li> <a href="http://news.gc.ca/web/fd-en.do?mthd=dpt&amp;ft=atom&amp;crtr.dpt1D=6675" rel="external">Government of Canada News Releases</a> </li>
-					</ul>
-					<p class="text-right"><strong><a href="#">All news</a></strong></p>
-				</div>
-			</div>
-		</section>
-	</div>
 </div>
-<section>
-	<h2>Recent activities</h2>
-	<div class="row">
-		<section class="col-sm-4">
-			<h3>Initiatives</h3>
-			<ul class="lst-spcd">
-				<li><a href="#">[Initiatives 1]</a></li>
-				<li><a href="#">[Initiatives 2 with longer text to wrap]</a></li>
-				<li><a href="#">[Initiatives 3]</a></li>
-				<li><a href="#">[Initiatives 4 with longer text to wrap]</a></li>
-			</ul>
-		</section>
-		<section class="col-sm-4">
-			<h3>Speeches</h3>
-			<ul class="lst-spcd">
-				<li><a href="#">[Speech 1]</a></li>
-				<li><a href="#">[Speech 2 with longer text to wrap]</a></li>
-				<li><a href="#">[Speech 3]</a></li>
-				<li><a href="#">[Speech 4 with longer text to wrap]</a></li>
-			</ul>
-		</section>
-		<section class="col-sm-4">
-			<h3>Media advisories</h3>
-			<ul class="lst-spcd">
-				<li><a href="#">[Media advisory 1]</a></li>
-				<li><a href="#">[Media advisory 2 with longer text to wrap]</a></li>
-				<li><a href="#">[Media advisory 3]</a></li>
-				<li><a href="#">[Media advisory 4 with longer text to wrap]</a></li>
-			</ul>
-		</section>
-	</div>
-</section>
-<section>
-	<h2>Gallery</h2>
-	<div class="row">
-		<div class="col-lg-3 col-md-6 mrgn-bttm-lg">
-			<a href="#">
-				<figure>
-					<img src="img/250x250.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
-					<figcaption>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Earum, fugiat!</figcaption>
-				</figure>
-			</a>
-		</div>
-		<div class="col-lg-3 col-md-6 mrgn-bttm-lg">
-			<a href="#">
-				<figure>
-					<img src="img/250x250.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
-					<figcaption>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Earum, fugiat!</figcaption>
-				</figure>
-			</a>
+<section class="gc-features">
```

Comment:
Please use the latest pattern for "gc-features", Although I understand we need this page template to work with v9.4.1 of GCWeb.